ChangeLog

```diff
@@ -31,14 +31,18 @@
   + scheduler: require pre-/post-start notifications correctly.
 
 - Changes since Pacemaker-2.0.4
+  + Important security fixes for CVE-2020-25654
```

Comment:
I'd probably put this as its own section after features. I'd describe it like "Prevent the bypassing of ACLs by direct IPC (CVE...)"